Makefile

```diff
@@ -55,6 +55,11 @@ endif
 SYMTABSRC = symtab_apps.c
 SYMTABOBJ = $(SYMTABSRC:.c=$(OBJEXT))
 
+# Turn off "Entering directory" messages as they mess up the output of
+# show-objs target.
+MAKEFLAGS += --no-print-directory
```

Comment:
show we add --no-print-directory to line 75 and remove this line to avoid the change to th default behaviour.